src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -5,9 +5,13 @@ import Tab from '@material-ui/core/Tab'
 import Tabs from '@material-ui/core/Tabs'
 import Typography from '@material-ui/core/Typography'
 import gql from 'graphql-tag'
+import Popover from '@material-ui/core/Popover'
+import Button from '@material-ui/core/Button'
 import { getIn } from 'immutable'
 import React, { useState } from 'react'
 import loadable from 'react-loadable'
+import SaveAltIcon from '@material-ui/icons/SaveAlt'
```

Comment:
Why did you choose to use this icon instead of the ordinary `Save` (floppy disc) icon? Maybe it's just me, but this icon makes me think that something is going to be downloaded, not saved when I see it